sdk/keyvault/Azure.Security.KeyVault.Administration/tests/BackupRestoreRecordedTestSanitizer.cs

```diff
@@ -14,28 +15,7 @@ public BackupRestoreRecordedTestSanitizer()
             : base()
         {
             AddJsonPathSanitizer("$..token");
-        }
-
-        private const string RetryAfter = "Retry-After";
-
-        /// <summary>
-        /// A RegEx to isolate a SAS token value.
-        /// </summary>
-        private Regex SignatureRegEx = new Regex(@"([\x0026|&|?]sig=)([\w\d%]+)", RegexOptions.Compiled);
-
-        public override string SanitizeUri(string uri)
-        {
-            return SignatureRegEx.Replace(uri, $"$1{SanitizeValue}");
-        }
-
-        public override void SanitizeHeaders(IDictionary<string, string[]> headers)
-        {
-            if (headers.ContainsKey(RetryAfter))
-            {
-                headers[RetryAfter] = new[] { "0" };
-            }
-
-            base.SanitizeHeaders(headers);
+            UriRegexSanitizers.Add(UriRegexSanitizer.CreateWithQueryParameter("sig", SanitizeValue));
```

Comment:
@christothes just to double check we don't end up leaking, this should be fine, yes? I don't even see any URIs with "sig" in the query params so it's hard to make sure there's no leaks, but since there's also no .json files in here and the tests pass, it seems safe.

Reply:
Yeah I didn't see any usages either, but figured I would add it just to be safe.